predicthq/endpoints/v1/beam/schemas.py

```diff
@@ -3,6 +3,28 @@
 from predicthq.endpoints.schemas import ArgKwargResultSet
 from typing import Optional, List
 
+# Python < 3.11 does not have StrEnum in the enum module
+import sys
+if sys.version_info < (3, 11):
+    import enum
+
+    class StrEnum(str, enum.Enum):
+        pass
+else:
+    from enum import StrEnum
+
+# Python < 3.9 does not have Annotated in the enum module
+if sys.version_info < (3, 9):
+    from typing_extensions import Annotated
+else:
+    from typing import Annotated, Literal
```

Comment:
do you need to import `Literal` again here since you handled it further down on lines 23-26